content/css/concepts/borders/terms/border-left/border-left.md

```diff
@@ -22,30 +22,53 @@ Shorthand property that defines the width, color, and style of the left border o
 border-left: <value>;
 ```
 
-where `<value>` can be one of the following:
+By default, `border-left` will have a style of `none` and a width of `medium`.
+The default color is the value of the `color` property or text color.
 
-- Border width: `thick`
-- Border style: `dashed`
-- Border color: `#f1f1f1`
+`<value>` can be one of the following:
 
-**Note:** values can be provided in any order.
+- `border-style` changes the shape or design of the border to one of the 10 available styles.
+- `border-width` changes the thickness of the border.
+- `border-color` changes the border color.
+
+> **Note:** values can be provided in any order.
 
 ## Example 1
 
 Set the left border of the `h1` element to be green, 3 pixels thick, and dotted:
 
 ```css
 h1 {
+  width: fit-content;
   border-left: green 3px dotted;
 }
 ```
 
+![Example 1](https://github.com/SanchitMahajan236/Codeacademy_docs/assets/116559051/2fe25faa-9e11-4095-a0e7-cb71ce6a7a39)
+
 ## Example 2
 
-Set the left border of the `h1` element to be 4 pixels thick, dashed, and blue:
+Set the left border of the `h1` element to be 3 pixels thick, dashed, and blue:
 
 ```css
 h1 {
-  border-left: 4px dashed blue;
+  width: fit-content;
+  border-left: 3px dashed #0000ff;
 }
 ```
+
+![Example 2](https://github.com/SanchitMahajan236/Codeacademy_docs/assets/116559051/fcd95df2-78c2-4f57-9ca1-7b42334110cb)
```

Comment:
You need to follow the instructions for file hosting as described in the style guide [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md).